.github/workflows/release.yml

```diff
@@ -95,8 +93,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer as an environment variable.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
+        # See https://www.packer.io/guides/hcl/variables under the
+        # "From environment variables" section for details about
+        # PKR_VAR environment variables
         run: |
+          PKR_VAR_winrm_password=\
```

Comment:
Same change as in `build.yml` but the variable should be added alphabetically.

Reply:
@mcdonnnj resolved with [e3b38fe](https://github.com/cisagov/windows-server-packer/pull/10/commits/e3b38fee3199914b48256f154ebff8382bc9701d)